hostwatch/hostwatch.go

```diff
@@ -43,3 +49,25 @@ func StartHostLabelSync(interval int, kClient *kubernetesclient.Client) error {
 	metadataClient.OnChange(interval, h.syncHostLabels)
 	return nil
 }
+
+// StartHostStatusSync ...
+func StartHostStatusSync(interval int, kClient *kubernetesclient.Client) error {
+	metadataAddress := os.Getenv("RANCHER_METADATA_ADDRESS")
```

Comment:
Don't we already read/initiate metadata client somewhere? Can't the client be re-used? Similar how we reuse k8s client